ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.9"
+    namespace: ocp
+    tag: openstack-installer
+  upi-installer:
+    name: "4.9"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate:
+    prerelease:
+      product: ocp
+      version_bounds:
+        lower: 4.10.0
+        upper: 4.11.0-0
+  latest:
+    prerelease:
+      product: ocp
+      version_bounds:
+        lower: 4.9.0
+        upper: 4.10.0-0
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.11"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi
+  cron: 0 18 */3 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.10 and @aws-ipi'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.10 and @aws-ipi'
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-provision
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/override-upgrade
```

Comment:
How about renaming `override-upgrade` to `upgrade-path`

Reply:
Sure. Let me update. Thanks!